src/main/java/core/mate/academy/model/Truck.java

```diff
@@ -1,13 +1,29 @@
 package core.mate.academy.model;
 
-/**
- * Add some custom fields that could be only in Truck
- * Do not remove no-field constructor
- */
 public class Truck extends Machine {
```

Comment:
The class implementation should not begin with an empty line. Please remove the empty line at the beginning of the class definition.